### PR TITLE
NA Style Refactor

### PR DESCRIPTION
**Pull request by outstandy**
Opening this PR to make sure it's on track:
🎨 Applying NA branding & styles to the home index
♻️ Removing unused & conflicting JS, sass, and markup
🔠 Switching to more concise/semantic naming conventions

WIP/TODO: 
🎨 Cleaning up & formatting project details & about pages
📱 More responsive behabior

### CODE REVIEW

**Comment by marklerner** [comment]
@outstandy Do you want me to wait on reviewing this PR until you've finished the WIP items?

**Comment by outstandy** [comment]
@marklerner whoops was missing a commit, pushed up—yeah my main concern is if there were any issues i might have missed stemming from removing the USWDS JS (which as far as I could tell was just handling some nonexistent input/form logic)

assuming that's all good i'll finish this up then ping you again.

**Comment by marklerner** [comment]
@outstandy So I think that the USWDS is also providing usa-grid, which we're using for structure on the page. This might just be a wrapper around some other grid system that we can pull in separately, but that is fairly key to the layout. That's the main thing I would look out for as you rejigger things.

Also, I think it's totally fine to keep the USWDS in there, if it works. We can always override parts of it that don't work for us.

**Comment by outstandy** [comment]
@marklerner we might want to leave all the USWDS stuff out if we're not basing it off that—the volume of overwrites/conflicts it creates are already making it hard to read & build on.

All the WIP stuff is applied, whenever you have a moment to spin it up and take a look.

**Comment by marklerner** [comment]
@outstandy This is so beautiful it makes me want to cry. THANK YOU SO MUCH!!! This is a potentially dumb request but could we make it so that the html file styling isn't as styled in as jagged a fashion? Just want to keep things consistent, and the styling looks like it splits in odd places (especially on the liquid template tags).

Also, it seems like when I use it on the mobile view on my laptop browser, I can't side-scroll on the table. Is that something you ran into?

File: _sass/variables.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
-
 // Typography
-$font-sans: 'Platform', 'Circular', 'Lyon', 'Arial', 'Palatino', sans-serif;
+$font-sans: "Platform", "Circular", "Lyon", "Arial", "Palatino", sans-serif;
```
Comment (marklerner) [inline _sass/variables.scss]: Doesn't this contradict what's in variables--na.sass? Duplicate variables.

File: _sass/variables--na.sass [hunk 2]
```diff
@@ -1,3 +1,6 @@
 $color-teal: #2fbcb3
-
+$color-beige: #f7f4f2
 $color-chart-background: #f5f5f5
+
+$font-sans: 'Circular', 'Helvetica', sans-serif
```
Comment (marklerner) [inline _sass/variables--na.sass]: Doesn't this duplicate what's in variables.scss?

File: _sass/layouts.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
+.open-covid-listing ul {
```
Comment (marklerner) [inline _sass/layouts.scss]: Be sure to change the class names to match the repo name "pandemic-response-repository". If these rely on autogenerated class names, they won't catch.

**Comment by outstandy** [comment]
@marklerner mobile table scroll is addressed 📜 📱

File: _sass/layouts.scss [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.layout--page main {
+	max-width: 1024px;
+	margin: 0 auto;
+	box-sizing: border-box;
+	padding: 60px 25% 60px 15px;
+}
+
+.layout--page.layout--open-covid main {
```
Comment (marklerner) [inline _sass/layouts.scss]: Same as above – layout--pandemic-response-repository

File: index.html [hunk 5]
```diff
@@ -1,23 +1,30 @@
 ---
 permalink: /
-title: Pandemic Response Repository
```
Comment (marklerner) [inline index.html:3]: This should stay as Pandemic Response Repository

**Review by marklerner — COMMENTED** [review]
Review is complete, take a look! Again, thank you so much for your help with this, it's amazing

**Comment by outstandy** [comment]
Addressed! Thanks @marklerner—lost track of my own overwrites & rebases 🙂 Let me know how this looks!

File: _sass/variables--na.sass [hunk 6]
```diff
@@ -1,3 +1,3 @@
 $color-teal: #2fbcb3
-
-$color-chart-background: #f5f5f5
+$color-beige: #f7f4f2
```
Comment (marklerner) [inline _sass/variables--na.sass:2]: This is also replicated in variables.scss but that's fine, they're identical.

**Review by marklerner — APPROVED** [review]
lgtm!

**Comment by marklerner** [comment]
Screen Shot 2020-04-07 at 4 05 23 PM

Looks like there's a weird bar at the bottom of the table. I'm not sure what it's from. Not launch blocking but figured I'd share.

**Pull request merged**